### DISCUSSION TITLE
Using an .exe file with command .vcpkg create?

### DISCUSSION THREAD

**Discussion by calicoder77** [microsoft/vcpkg#28306] [Q&A] [answered]
Good day all, wondering what advice you can give regarding using the .\vcpkg create command for a website that only offers an .exe as a download. Can I simply extract that exe onto my local folder and zip it up and pass that to the next step .\vcpkg build?

Thanks in advance for any help, much appreciated!

**Comment by JackBoosY** [comment, marked as answer]
Well, it should work and you won't loss the signature verification.

**Reply by calicoder77** [reply]
Excellent thanks and good to know. Does vcpkg create support downloading from URL with extra data in the download link like & signs? I'm getting an error that the ampersand is reserved for future use.

Update: fixed by adding "" to each ampersand.

**Reply by JackBoosY** [reply]
> Does vcpkg create support downloading from URL with extra data in the download link like & signs?

Not likely. vcpkg will use curl or some other tool to download the file directly without adding any other data.

As you know, modify / delete or add extra data will break the signature.
